calcom/cal.com
# fix: Unordered bookings

## What does this PR do?
FIxes the issue of unordered bookings happened after bookings query optimization.

[Before and After Demo](https://www.loom.com/share/f43e379bc17041b1ab54f0c1646e9ac2)

## Type of change

## How should this be tested?
- Book the `teampro` user by some guest email for Date-2
- Book the `free` user by` teampro@example.com` email(So `teampro` becomes attendee) for Date-1. Date-1 is earlier than Date-2
- Login to `teampro` and check the bookings. Notice that Date-2 booking is showing earlier than Date-1. This bug should be fixed in this PR. (It happened because attendee based bookings and organizer bookings are queried independently and concatenated.)

## Mandatory Tasks

## Checklist
- I haven't added tests that prove my fix is effective or that my feature works

## Followup
- [ ] Test to avoid this regression #10950
